keep/api/core/db.py

```diff
@@ -489,6 +503,16 @@ def add_or_update_workflow(
         return existing_workflow if existing_workflow else workflow
 
 
+def get_workflow_to_alert_execution_by_workflow_execution_id(
+    workflow_execution_id: str
```

Comment:
add tenant_id filtering

Reply:
In this function I am directly querying `workflowtoalertexecution` table which doesn't have tenant_id column, if we really like to filter by tenant id, we need to query workflowexecution and then query workflowtoalert execution but I am not sure if it is really necessary. What do you think about it?